### PR TITLE
Align Changelog generator across the repos

### PR DESCRIPTION
**Pull request by micbar**
## Description
- Make it possible to run CI on forked repos
- Update to calens v0.2.0
- Update the template

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -397,7 +397,11 @@ def binary(ctx, name):
           'files': [
             'dist/release/*',
           ],
-          'title': ctx.build.ref.replace("refs/tags/", ""),
+          'checksum': [
```
Comment (tboerger) [inline .drone.star]: You should drop that, checksum files are already part of the Makefile, and they should stay within the Makefile as they are also uploaded to our mirrors, not only GitHub releases.

File: .drone.star [hunk 2]
```diff
@@ -397,7 +397,11 @@ def binary(ctx, name):
...
+            'md5',
+            'sha256'
+          ],
+          'title': ctx.build.ref.replace("refs/tags/v", ""),
```
Comment (tboerger) [inline .drone.star]: This should also be reverted, IMHO it's totally fine to keep the `v` prefix for the release title.

File: .drone.star (hunk 1 above)
Comment (micbar) [reply .drone.star]: ok, didn't know that. will drop it.

File: .drone.star (hunk 2 above)
Comment (micbar) [reply .drone.star]: If we compare with the Releases of other Repos, we don't include the `v` there.
We are using the `v` prefix only for the tag itself. Referring to Semver, a release Version has no prefix, but the tag has. What do you think?

In the end maybe a matter of taste?

Comment (tboerger) [reply .drone.star]: Than you got to see what others say. I personally like the v prefix :)

File: .drone.star (hunk 1 above)
Comment (tboerger) [reply .drone.star]: If I can add a private GPG key to Drone we could also automatically generate `.asc` files to have something more trustable than checksum failes.

File: .drone.star (hunk 2 above)
Comment (micbar) [reply .drone.star]: There are strong opinios against `v` in Release titles 🤷‍♂

**Comment by micbar** [comment]
Build is failing because Drone is not using the latest calens code from master.

@tboerger Should we do it like in core, use the toolhippie/calens image?

**Comment by tboerger** [comment]
> Build is failing because Drone is not using the latest calens code from master.

As said at lunch time, patch the makefile to something like `go run github.com/restic/calens@master >| CHANGELOG.md` for the changelog task.

**Comment by micbar** [comment]
Sample output
--------------
# Changelog for [0.1.0] (2020-01-24)

The following sections list the changes in ownCloud ocis-hello 0.1.0.

[0.1.0]: https://github.com/owncloud/ocis-hello/compare/c43f3a33cb0b57d7e25ebc88c138d22e95f88cfe...v0.1.0

## Summary

 * Chg #2: Initial release of basic version

## Details

 * Change #2: Initial release of basic version

   Just prepared an initial basic version to serve a hello world API that also provides a Phoenix
   extension to demonstrate the plugin architecture in combination with Phoenix and ownCloud
   Infinite Scale.

   https://github.com/owncloud/ocis-hello/issues/2

File: changelog/CHANGELOG.tmpl [hunk 3]
```diff
@@ -1,7 +1,24 @@
...
-# Changelog for {{ .Version }}
+{{ $allVersions := . }}
+{{- range $index, $changes := . }}{{ with $changes -}}
+# Changelog for [{{ .Version }}] ({{ .Date }})
 
-The following sections list the changes for {{ .Version }}.
+The following sections list the changes in ownCloud ocis-hello {{ .Version }}.
+
+{{/* creating version compare links */ -}}
+{{ $next := add1 $index -}}
```
Comment (micbar) [inline changelog/CHANGELOG.tmpl]: @tboerger This will add version compare links

**Comment by micbar** [comment]
And I create a v0.1.0 folder

**Comment by micbar** [comment]
@tboerger removed the second template

**Pull request merged**